math/minuit2/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@
 
 cmake_minimum_required(VERSION 3.1)
 
+if(NOT CMAKE_PROJECT_NAME STREQUAL ROOT)
+  project(Minuit2 VERSION ${ROOT_VERSION} LANGUAGES CXX)
```

Comment:
```suggestion
  project(Minuit2 LANGUAGES CXX)
```

ROOT_VERSION is not defined if this is not ROOT (the line above). You can't easily get the project version here; hopefully it is safe to redefine later.